common/common.gradle

```diff
@@ -10,12 +10,15 @@ sourceCompatibility = 1.11
 targetCompatibility = 1.11
 
 dependencies {
-    compileOnly(group: 'com.google.guava', name: 'guava', version: '28.2-jre')
-    compileOnly(group: 'org.springframework.boot', name: 'spring-boot', version: '2.2.5.RELEASE')
-    compileOnly(group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: '2.11.2')
-    compileOnly(group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2')
+    compileOnly(enforcedPlatform("io.spinnaker.clouddriver:clouddriver-bom:${clouddriverVersion}"))
```

Comment:
What's the purpose of this dependency?

Reply:
I believe is to use transitive dependency versions from Clouddriver
same as `clouddriver/clouddriver.gradle` https://github.com/armory-plugins/external-accounts/blob/a7f7f96e448a6b44bb046a97cdcd47b38c4bd38f/clouddriver/clouddriver.gradle#L25